<?xml version="1.0" ?>
<sdf version="1.6">
    <model name="conveyor_belt">
        <static>true</static>
        <link name="belt">
            <sensor name='conveyor_belt_contact' type='contact'>
                <update_rate>10</update_rate>
                <contact>
                    <collision>belt_collision</collision>
                </contact>
                <plugin name="conveyor_belt_plugin" filename="libROSConveyorBeltPlugin.so">
                    <belt_speed>0.2</belt_speed>
                </plugin>
            </sensor>
            <inertial>
                <mass>10</mass>
                <inertia>
                    <ixx>20.841</ixx>
                    <ixy>0</ixy>
                    <ixz>0</ixz>
                    <iyy>0.8417</iyy>
                    <iyz>0</iyz>
                    <izz>21.66667</izz>
                </inertia>
            </inertial>
            <collision name="belt_collision">
            <geometry>
                <box>
                <size>1 5 0.1</size>
                </box>
            </geometry>
            <surface>
                <friction>
                    <ode>
                        <mu>0.0</mu>
                        <mu2>0.0</mu2>
                    </ode>
                </friction>
            </surface>
            </collision>
            <visual name="visual">
            <geometry>
                <box>
                <size>1 5 0.1</size>
                </box>
            </geometry>
            </visual>
        </link>
    </model>
</sdf>
